### PR TITLE
Fixing layout problems in SearchResult component

### PR DESCRIPTION
**Pull request by Klynger**
#### What is the purpose of this pull request?
Align elements in SearchResult component

#### What problem is this solving?
Layout problems and the missing searched term in mobile view

#### How should this be manually tested?
[Access this workspace](https://searchlayout--storecomponents.myvtex.com/Samsung/s?map=ft) in mobile view
#### Screenshots or example usage
image

#### Types of changes

* [x] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/global.css [hunk 1]
```diff
@@ -133,10 +133,14 @@ body.vtex-filter-popup-open {
   }
 }
 
-@media only screen and (min-width: calc(40em + 1px)) {
+@media only screen and (min-width: 40em) {
   .vtex-search-result__border {
     display: none;
   }
+
+  .vtex-search__term {
```
Comment (estacioneto) [inline react/global.css]: With tachyons you can do `dn-ns`

File: react/components/SearchResult.js [hunk 2]
```diff
@@ -47,12 +47,17 @@ export default class SearchResult extends Component {
       hiddenFacets,
```
Comment (estacioneto) [inline react/components/SearchResult.js]: CHANGELOG.md?

File: react/components/SearchResult.js [hunk 3]
```diff
@@ -47,12 +47,17 @@ export default class SearchResult extends Component {
...
     } = this.props
 
+    const term = params && params.term ? params.term : null
```
Comment (estacioneto) [inline react/components/SearchResult.js]: Why do you need the final `?  params.term : null` ?

File: react/global.css (hunk 1 above)
Comment (Klynger) [reply react/global.css]: Because this is inside a breakpoint, so I can't make it ```display: none``` only with the class ```dn-ns``` from tachyons

**Comment by luciannojunior** [comment]
@oldufre please, check this out

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Change breakpoint of vertical divider in SearchResult in mobile view and add the searched term to the view
```
Comment (lucasecdb) [inline CHANGELOG.md]: Update to
```md
### Fixed
- Change breakpoint of vertical divider in SearchResult in mobile view and add the searched term to the view
```

**Pull request merged**